Docs: generate CLI command reference

reinkrul opened the pull request:
See https://nuts-node.readthedocs.io/en/docs-cli-command-reference/pages/deployment/cli/index.html

gerardsn commented on docs/pages/deployment/cli/commands/nuts_network.rst:
```diff
@@ -0,0 +1,34 @@
+.. _nuts_network:
```
```
nuts network -h
```
returns
```
network commands

Usage:
  nuts network [command]

Available Commands:
  get         Gets a transaction from the network
  list        Lists the transactions on the network
  payload     Retrieves the payload of a transaction from the network
  peers       Get diagnostic information of the node's peers
  reprocess   Reprocess all transactions with the give contentType (ex: application/did+json)

Flags:
      --address string     Address of the remote node. Must contain at least host and port, URL scheme may be omitted. In that case it 'http://' is prepended. (default "localhost:1323")
  -h, --help               help for network
      --timeout duration   Client time-out when performing remote operations, such as '500ms' or '10s'. Refer to Golang's 'time.Duration' syntax for a more elaborate description of the syntax. (default 10s)
      --verbosity string   Log level (trace, debug, info, warn, error) (default "info")
```

The `Available Commands` section from the help are in the docs replaced by the `SEE ALSO` section and moved to the end. To me it was not clear that this section contains the available commands and looks more like a suggested reading section that I can ignore. Can this be changed to look a bit more like the help?

woutslakhorst commented:
<img width="294" alt="Screenshot 2022-07-08 at 11 16 38" src="https://user-images.githubusercontent.com/703298/177960468-c97fca7b-8c18-4596-95e1-d12aaaf2b396.png">

The menu options are a bit overkill. I think that each command can do without the submenu

woutslakhorst reviewed (changes requested):
see above comment

gerardsn commented on docs/pages/deployment/cli/commands/nuts_network.rst:
```diff
@@ -0,0 +1,34 @@
+~~~~~~~~
+
+
+network commands
```
shouldn't this be `network [command]`

gerardsn commented on docs/pages/deployment/cli/commands/nuts_network_get.rst:
```diff
@@ -0,0 +1,39 @@
+
+::
+
+  nuts network get [ref] [flags]
+
+Options
+~~~~~~~
+
+::
+
+  -h, --help   help for get
+
+Options inherited from parent commands
```
- Inconsistent use of `[flags]` and `options` is confusing.
- Not all options/flags/inputs are listed, but this is probably more a documentation thing.

reinkrul replied:
out of scope

gerardsn reviewed (dismissed):
Great addition to the docs.

I think the way the information presented by the cli help is clearer than the docs though.

I've made some comments on the network command, but I think they apply everywhere.

reinkrul commented:
Removed See Also section, joined flags/options because "inherited from parent" didn't add anything. Changed the subsections to bolded text, so they don't show up in the TOC. Added horizontal lines between commands.
